docs/case-studies/jenkins-cicd.md

```diff
@@ -2,19 +2,19 @@
 title:  Jenkins + KubeVela for CI/CD
 ---
 
-This section will introduce how to combine KubeVela and Jenkins to do CI (continuous integration) and CD (continuous delivery). The following example is based on the publish process of a simple HTTP server. The code can be found at [here](https://github.com/Somefive/KubeVela-demo-CICD-app).
+This section will introduce how to combine KubeVela with Jenkins to do CI (continuous integration) and CD (continuous delivery). The following example is based on the release process of a simple HTTP server. The code can be found at [here](https://github.com/Somefive/KubeVela-demo-CICD-app).
 
 ## Preparation
 
 Before combining KubeVela and Jenkins, you need to ensure the following environments have already been set up.
 
 1. Deploy Jenkins service with Docker support, including related plugins and credentials which will be used to access image repository.
-2. A git repository with Webhook enabled. Ensure commits to the target branch will trigger the running of Jenkins pipeline.
-3. Get Kubernetes for deployment. Install KubeVela and enable its apiserver. Ensure the KubeVela apiserver can be accessed from external.
+2. A git repository with Webhook enabled. Ensure commits to the target branch will trigger the running of the Jenkins pipeline.
+3. Get Kubernetes for deployment. Install KubeVela and enable its apiserver. Ensure the KubeVela apiserver can be accessed from external environment.
 
 ## Combining Jenkins with KubeVela apiserver
 
-Deploy Jenkins pipeline with the following Groovy script. You can change the git repository address, image address, apiserver address and other environment configurations on demand. Your git repository should contain the Dockerfile and app.yaml configuration file to tell how to build target image and which component the application contains.
+Deploy Jenkins pipeline with the following Groovy script. You can change the git repository address, image address, apiserver address and other environment configurations on demand. Your git repository should contain the `Dockerfile` and `app.yaml` configuration file to tell how to build the target image and which component the application contains.
```

Comment:
Should the client be granted access to the API server?

Reply:
In blog which introduce it in depth, there is a comment for the apiserver credential issue. Currently, the apiserver do not have any user control system, so no access is needed for now. But this will lead to security issue, so we must fix it in later version.